src/internal/clusterstate/v2.5.0/collection.go

```diff
@@ -277,22 +313,41 @@ func migratePostgreSQLCollection(ctx context.Context, tx *pachsql.Tx, name strin
 					return err
 				}
 
-				updateFields := []string{}
-				for k := range params {
-					updateFields = append(updateFields, fmt.Sprintf("%s = :%s", k, k))
+				var (
+					col_values []any // Array with this row's values for update
+					w_values   []any // Array with the values for this row's WHERE clause
+				)
+
+				// Loop through each of the columns, get the value for update, and append to col_values
+				for _, k := range columns {
+
+					// If this is a proto field, we need to hex encode it because it's actually binary
+					if k == "proto" {
+						col_values = append(col_values, hex.EncodeToString([]byte(params[k].([]uint8))))
```

Comment:
byte and uint8 are the same type:
```go
package builtin
type byte = uint8
```
`params[k].([]byte)` should be adequate here.

Reply:
fixed